src/server/app.js

```diff
@@ -1,27 +1,30 @@
-const express = require('express');
-const logger = require('morgan');
-const path = require('path');
-const app = express();
+import express from 'express';
+import logger from 'morgan';
+import path from 'path';
+import routes from './routes';
 
-const loggerType = process.env.NODE_ENV === 'production' ? 'common' : 'dev'
-app.use(logger(loggerType));
+const App = express();
+
+const loggerType = process.env.NODE_ENV === 'production' ? 'common' : 'dev';
+App.use(logger(loggerType));
 
 // Accept CORS
-app.use(function(req, res, next) {
+App.use(function (req, res, next) {
```

Comment:
I see you used an arrow function so maybe consider changing this function to an arrow function as well to keep consistency.